.travis.yml

```diff
@@ -66,8 +68,8 @@ jobs:
 
   - stage: master
     script:
-      - yarn chromatic --quiet --auto-accept-changes
-      - yarn build-storybook # chromatic runs `build-storybook`, but puts it into a random directory. We just build it again
+      - yarn build-storybook # build storybok to .storybook and then we can use it in chromatic
```

Comment:
I think you could just remove this line. If you told `yarn chromatic` to build into the same directory as `build-storybook`, this will still build twice, just in the same directory (unless Storybook bails early if already built?)